src/components/Dashboard/__snapshots__/Dashboard.story.storyshot

```diff
@@ -1260,10 +1262,12 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT/Dashb
                         open={false}
                         tabIndex={0}
                         title="Select time range"
+                        type="button"
```

Comment:
Not sure if this was added by us, but its redundant

Reply:
Intentional change from Carbon: https://github.com/carbon-design-system/carbon/pull/6612